test/views/ConfirmWebsiteViewSpec.scala

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import views.html.ConfirmWebsiteView
+
+class ConfirmWebsiteViewSpec extends ViewBaseSpec {
+
+  val injectedView: ConfirmWebsiteView = injector.instanceOf[ConfirmWebsiteView]
+
+  object Selectors {
+    val heading = "heading-large"
+    val backLink = "#content > article > a"
+    val continueButton = ".button"
+    val editLink = "#content > article > p:nth-child(5) > a"
+  }
+
+  "The Confirm Website view" should {
+    lazy val view = injectedView(testWebsite)
+    lazy implicit val document: Document = Jsoup.parse(view.body)
+
+    "have the correct heading" in {
+      document.getElementsByClass(Selectors.heading).text() shouldBe "Confirm the website address"
+    }
+
+    "have a back link" which {
+
+      "should have the correct text" in {
+        elementText(Selectors.backLink) shouldBe "Back"
+      }
+
+      "should have the correct back link" in {
+        element(Selectors.backLink).attr("href") shouldBe controllers.routes.CaptureWebsiteController.show().url
+      }
+    }
+
+    "have the website address the user provided" in {
```

Comment:
This test seems a bit unspecific. There's currently also no test for the paragraph containing the website, so it'd make more sense to test that specific paragraph text which would also test that the website is played back.